docs/EC-UI-screenshot.md

```diff
@@ -0,0 +1,142 @@
+# EC WEB-UI Screenshots as per Tabs
+ * [1.View](#1_view)
```

Comment:
please use proper markdown numeric indent example here-

https://stackoverflow.com/questions/18088955/markdown-continue-numbered-list

@vidya22022 @Subhash-vishwakarma

Reply:
yeah Sure @ayasuda-ge will try to close this today

Reply:
using numeric markdown we got this solution only else we can use bullets markdown @ayasuda-ge

Reply:
@ayasuda-ge  By this solution we mean using of &nbsp; for space there / for new line

Reply:
Consider this @Subhash-vishwakarma @vidya22022

1. this is the proper mardown
  1.1 another market down
  1.2 next mark down
  1.3 blah blah
2. final markdown
3. summary

Reply:
@ayasuda-ge  We marked index with above format but in raw you we have used & &nbsp for space we also tried with \ but its not working so is that fine to use &nbsp? or we can go with bullets also one more question i have i created 1 KT document in different file do you want me to add on same screenshot file or its keep as it is separate ?